src/map/conquest_data.cpp

```diff
@@ -88,6 +90,13 @@ uint8 ConquestData::getRegionOwner(REGION_TYPE region) const
 {
     uint8 regionNum = static_cast<uint8>(region);
 
+    // Cities (including Jeuno/Tavnazia), Dynamis/Limbus, Lumoria (sea) are all "outside of own nation's control".
+    // Even if the character belongs to the nation.
+    if (zoneutils::IsAlwaysOutOfNationControl(region))
+    {
+        return 4;
```

Comment:
I would avoid using a regionId 4. This locks us into always having 3 regions. That may be fine for Era / normal servers, but if someone wants to create a custom server with their custom region, it would be much harder to do so.

You could technically use something like uint8.max, but instead, I think it would be better to leave `getRegionOwner` alone. In the places where you call this and check for region owner == 4, call a new method `conquest::IsRegionAlwaysOutOfNationControl`. You will not need the zoneutils method if you do that.

Reply:
That being said, I am not sure how much LSB cares about making regions customizable.

Reply:
(spoiler alert: we're not interested :P)

Reply:
@xkoredev I went ahead and updated the logic to be a bit simpler. We can rely on the `or` operator short-circuiting the evaluation to avoid running into an out-of-bounds issue.